flowgate-common/src/main/java/com/vmware/flowgate/common/MetricKeyName.java

```diff
@@ -11,4 +11,13 @@
    public static final String PDU_XLET_CURRENT = "%s|Current";
    public static final String PDU_XLET_VOLTAGE = "%s|Voltage";
    public static final String PDU_XLET_FREE_CAPACITY = "%s|FreeCapacity";
+
+   public static final String SERVER_FRONT_TEMPERATURE_LOCATIONX = "FrontTemperature|Location:%s";
```

Comment:
You don't need to place the world location in the key.

Some thing like "FrontTemperature|%s" should be good.  What do you think?

Reply:
ok